packages/instantsearch-react/examples/Search.js

```diff
@@ -70,7 +75,12 @@ class Search extends Component {
             defaultSelectedIndex="instant_search"
           />
           <HitsPerPage items={[10, 20, 30]} defaultHitsPerPage={20} />
-          <Range attributeName="price" />
+          <Range
+            attributeName="price"
+            translations={{
+              value: v => `$${v.toLocaleString()}`,
```

Comment:
I'm trying to display the price as `$16`. The first `$` is the actual dollar sign, while the second one is there for  string interpolation.